homeassistant/components/weheat/sensor.py

```diff
@@ -95,6 +95,33 @@ class WeHeatSensorEntityDescription(SensorEntityDescription):
         suggested_display_precision=DISPLAY_PRECISION_WATER_TEMP,
         value_fn=lambda status: status.air_inlet_temperature,
     ),
+    WeHeatSensorEntityDescription(
+        translation_key="thermostat_water_setpoint",
+        key="thermostat_water_setpoint",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=DISPLAY_PRECISION_WATER_TEMP,
+        value_fn=lambda status: status.thermostat_water_setpoint,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="thermostat_room_temperature",
+        key="thermostat_room_temperature",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=DISPLAY_PRECISION_WATER_TEMP,
+        value_fn=lambda status: status.thermostat_room_temperature,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="thermostat_room_temperature_setpoint",
+        key="thermostat_room_temperature_setpoint",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=DISPLAY_PRECISION_WATER_TEMP,
+        value_fn=lambda status: status.thermostat_room_temperature_setpoint,
+    ),
```

Comment:
Does this only support 1 room?

Reply:
Yes, it is the room where the thermostat is placed. Dutch screenshot from the portal:
![image](https://github.com/user-attachments/assets/3e52d429-5493-454d-9352-4228a643ce20)